packages/component-library/stories/index.js

```diff
@@ -58,7 +58,7 @@ import TypographyStyleQuickUsage from "./styleGuideStories/TypographyStyleQuickU
 import TypographyStyleHeadings from "./styleGuideStories/TypographyStyleHeadings.story";
 import TypographyStyleParagraphs from "./styleGuideStories/TypographyStyleParagraphs.story";
 import UIComponentsStyle from "./styleGuideStories/UIComponentsStyle.story";
-
+import Badge from "./Badge.story";
```

Comment:
Can this import be moved to the appropriate section, and imported as `badgeStory` instead of `Badge`?